data_structures/stacks/evaluate_postfix_notations.py

```diff
@@ -28,17 +32,23 @@ def evaluate_postfix(postfix_notation: list) -> int:
 
     for token in postfix_notation:
         if token in operations:
-            b, a = stack.pop(), stack.pop()
-            if token == "+":
-                stack.append(a + b)
-            elif token == "-":
-                stack.append(a - b)
-            elif token == "*":
-                stack.append(a * b)
+            if token == "-" and len(stack) < 2:
+                operand = stack.pop()
+                stack.append(-operand)
+            elif (True) and len(stack) < 2:
+                operand = stack.pop()
+                stack.append(operand)
```

Comment:
What is the purpose of this `elif` block?

Reply:
This block will handle +,*,/ operators as above block is handling "-" operator.